setup.py

```diff
@@ -0,0 +1,49 @@
+import os
+import shutil
+import argparse
+import subprocess
+
+def clone_repo(workdir):
+    os.chdir(workdir)
+    subprocess.run(["git", "clone", "https://github.com/gnoswap-labs/gnoswap.git"], check=True)
+    os.chdir("gnoswap")
+
+def copy_contracts(workdir):
+    gno_dir = os.path.join(workdir, "gno", "examples", "gno.land")
+
+    # Copy GRC20 tokens
+    shutil.copytree("__local/grc20_tokens", os.path.join(gno_dir, "r", "demo"), dirs_exist_ok=True)
+    shutil.copytree("_deploy/r/gnoswap", os.path.join(gno_dir, "r", "demo"), dirs_exist_ok=True)
+
+    # Copy gnoswap base packages
+    shutil.copytree("_deploy/p/gnoswap", os.path.join(gno_dir, "p", "demo"), dirs_exist_ok=True)
+
+    # Copy gnoswap base realms
+    shutil.copytree("_deploy/r/gnoswap", os.path.join(gno_dir, "r", "demo", "gnoswap"), dirs_exist_ok=True)
```

Comment:
```suggestion
    shutil.copytree("_deploy/r/gnoswap", os.path.join(gno_dir, "r", "gnoswap"), dirs_exist_ok=True)
```
We removed `demo` in our path for testnet4 right?